src/js/map/choropleth/subindicator_calculator.js

```diff
@@ -2,25 +2,19 @@ import {percFmt, numFmt} from '../../utils'
 
 export default class SubindicatorCalculator {
     calculate(args) {
-        if (typeof args.data === 'undefined'){
+        if (typeof args.data === 'undefined') {
             return null;
         }
 
-        const result = Object.entries(args.data).map(childGeography => {
-            const code = childGeography[0];
-            const count = childGeography[1];
-            const universe = args.subindicatorArr.reduce((el1, el2) => {
-                if (el2.children != undefined && el2.children[code] != undefined) {
-                    let e1 = typeof el1 === 'undefined' ? 0 : el1;
```

Comment:
this check was unnecessary because e1 is the total and can never be `undefined`. it starts with `0` the value we call `reduce` with and then is always the return value from the method itself.